riscv/debug_module.cc

```diff
@@ -43,7 +43,7 @@ debug_module_t::debug_module_t(simif_t *sim, const debug_module_config_t &config
   // them because I'm too lazy to add the code to just ignore accesses.
   hart_state(1 << field_width(sim->get_cfg().max_hartid() + 1)),
   hart_array_mask(sim->get_cfg().max_hartid() + 1),
-  rti_remaining(0)
+  rti_remaining(0), hart_available_state{true, true}
```

Comment:
Seems clunky that the array's size of 2 is repeated multiple times in this code.  If you invert the polarity and make this `hart_unavailable_state` instead, you could use the initializer `{false}` to do the right thing regardless of size.